lunes_cms/cms/templates/admin/base.html

```diff
@@ -17,5 +17,19 @@
             // Fix position of counter when sidebar is collapsed
             feedbackNavItem.find("p").width("auto");
         }
+        toggleFields($('#id_word_type').val())
+        $('#id_word_type').change(function(){
+            toggleFields($(this).val())
+        })
+
+        function toggleFields(wordType) {
+            if (wordType == "Nomen") {
+                $(".fieldBox.field-plural")[0].hidden = false
+                $("label[for='id_plural']")[0].hidden = false
+            } else {
+                $(".fieldBox.field-plural")[0].hidden = true
+                $("label[for='id_plural']")[0].hidden = true
+            }
```

Comment:
```suggestion
            $("#id_plural").closest("div.row").toggle(wordType == "Nomen");
```
I'd toggle the whole row (including the plural article), not only the plural field.

Reply:
First I just did it for the entire row, but since we still have an ongoing discussion about whether or not we even need the plural article, I only did it for plural